### DISCUSSION TITLE
Typesense hostname hardcoded?

### DISCUSSION THREAD

**Discussion by stumpylog** [immich-app/immich#2122] [Q&A] [answered]
Is there no environment variable to set the hostname of typesense?  Something like the REDIS_HOSTNAME and DB_HOSTNAME that already exist.

**Comment by alextran1502** [comment, marked as answer]
Yes, there is `TYPESENSE_HOSTNAME`. The documentation needs some love after the barrage of changes and new features in the last couple weeks

**Reply by stumpylog** [reply]
No worries!  I'm a few versions behind in updating my compose anyway.  Thank you!

**Reply by alextran1502** [reply]
Correction, it should be `TYPESENSE_HOST` :D

![image](https://user-images.githubusercontent.com/27055614/228705964-54cfc1fc-a6ba-4f26-a0fa-91538fd97076.png)
